ample/util/mrbump_util.py

```diff
@@ -258,6 +279,23 @@ def _extractResults(self, mrbump_dir, archived_ensembles=None):
         self.results = results
         return
 
+    @staticmethod
+    def jobSucceeded(job_dict):
```

Comment:
`job_succeeded`

Reply:
Yes. I''ll put cleaning up the mess of case in this file as an issue